ocrd_network/ocrd_network/deployer.py

```diff
@@ -229,6 +231,23 @@ def deploy_rabbitmq(
             remove: bool,
             ports_mapping: Union[Dict, None] = None
     ) -> str:
+        if self.data_queue.skip_deployment:
+            self.log.debug(f"RabbitMQ is externaly managed. Skipping deployment")
+            verify_rabbitmq_available(
+                self.data_queue.address,
+                self.data_queue.port,
+                self.data_queue.vhost,
+                self.data_queue.username,
+                self.data_queue.password
+            )
+            wait_for_rabbitmq_availability(
+                host=self.data_queue.address,
+                port=int(self.data_queue.port),
+                vhost='/',
+                username=self.data_queue.username,
+                password=self.data_queue.password
+            )
```

Comment:
Why do we need both of these availability checks? Isn't `wait_for_rabbitmq_availability` already performing the required check? I think it does. I like the new naming convention - `verify_rabbitmq_available`. It just has to do multiple checks to simulate the timeout just like in `wait_for_rabbitmq_availability`. Then we can get rid of the `wait_for_rabbitmq_availability` method.

Reply:
Thanks, updated accordingly